cctrl/user_commands.py

```diff
@@ -72,6 +72,30 @@ def parse_cmdline(user):
         help="the password")
     create_subparser.set_defaults(func=user.create)
 
+    setup_subparser = subparsers.add_parser('setup', help="setup user")
+    setup_subparser.add_argument(
+        "--email",
+        nargs='?',
```

Comment:
yes we want, they don't have to be set https://github.com/cloudControl/documentation/blob/sshauth/Platform%20Documentation.md#setup-cli